### DISCUSSION TITLE
Deepgram connection failed

### DISCUSSION THREAD

**Discussion by csefe** [deepgram/community#837] [General help]
While using livekit deepgram plugin, I am getting this error in Docker container.
Base Docker image: 
FROM  osaiai/dokai:24.06-cpu.ffmpeg

{"message": "deepgram connection failed, retrying in 0s", "timestamp": "2024-07-04T10:33:02.295722+00:00"}
{"message": "deepgram connection failed, retrying in 2s", "timestamp": "2024-07-04T10:33:02.298708+00:00"}
{"message": "deepgram connection failed, retrying in 0s", "job_id": "X", "pid": 26, "timestamp": "2024-07-04T10:33:02.298776+00:00"}
{"message": "deepgram connection failed, retrying in 2s", "job_id": "X", "pid": 26, "timestamp": "2024-07-04T10:33:02.299223+00:00"}
{"message": "deepgram connection failed, retrying in 4s", "timestamp": "2024-07-04T10:33:04.299169+00:00"}
{"message": "deepgram connection failed, retrying in 4s", "job_id": "X", "pid": 26, "timestamp": "2024-07-04T10:33:04.299546+00:00"}
{"message": "deepgram connection failed, retrying in 6s", "timestamp": "2024-07-04T10:33:08.300440+00:00"}
{"message": "deepgram connection failed, retrying in 6s", "job_id": "X", "pid": 26, "timestamp": "2024-07-04T10:33:08.300850+00:00"}

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by csefe** [reply]
My Code(copied from https://docs.livekit.io/agents/quickstart/)

import asyncio
import logging

from livekit.agents import JobContext, JobRequest, WorkerOptions, cli
from livekit.agents.llm import (
    ChatContext,
    ChatMessage,
    ChatRole,
)
from livekit.agents.voice_assistant import VoiceAssistant
from livekit.plugins import deepgram, elevenlabs, openai, silero

# This function is the entrypoint for the agent.
async def entrypoint(ctx: JobContext):
    # Create an initial chat context with a system prompt
    initial_ctx = ChatContext(
        messages=[
            ChatMessage(
                role=ChatRole.SYSTEM,
                text="You are a voice assistant created by LiveKit. Your interface with users will be voice. Pretend we're having a conversation, no special formatting or headings, just natural speech.",
            )
        ]
    )

    # VoiceAssistant is a class that creates a full conversational AI agent.
    # See https://github.com/livekit/agents/blob/main/livekit-agents/livekit/agents/voice_assistant/assistant.py
    # for details on how it works.
    assistant = VoiceAssistant(
        vad=silero.VAD(), # Voice Activity Detection
        stt=deepgram.STT(), # Speech-to-Text
        llm=openai.LLM(), # Language Model
        tts=elevenlabs.TTS(), # Text-to-Speech
        chat_ctx=initial_ctx, # Chat history context
    )

    # Start the voice assistant with the LiveKit room
    assistant.start(ctx.room)

    await asyncio.sleep(3)

    # Greets the user with an initial message
    await assistant.say("Hey, how can I help you today?", allow_interruptions=True)


# This function is called when the worker receives a job request
# from a LiveKit server.
async def request_fnc(req: JobRequest) -> None:
    logging.info("received request %s", req)
    # Accept the job tells the LiveKit server that this worker
    # wants the job. After the LiveKit server acknowledges that job is accepted,
    # the entrypoint function is called.
    await req.accept(entrypoint)


if __name__ == "__main__":
    # Initialize the worker with the request function
    cli.run_app(WorkerOptions(request_fnc))

**Comment by jpvajda** [comment]
@csefe  are you still having this issue? if so can you provide some request ids so I can take a deeper look?
